router/navigation-drawer.tsx

```diff
@@ -132,11 +133,16 @@ export const NavigationDrawer: React.FC<NavDrawerProps> = ({ navigation }) => {
     );
 
     return (
-        <Drawer activeItem={selected} onItemSelect={(id: string): void => selectItem(id)}>
+        <Drawer
+            activeItem={selected}
+            onItemSelect={(id: string): void => selectItem(id)}
+            theme={theme.dark ? blueDarkSupplement : {}}
+        >
             <DrawerHeader
                 title={'Drawer Title'}
                 subtitle={'Drawer Subtitle'}
                 backgroundImage={headerBgImage}
+                backgroundColor={Colors.blue[500]}
```

Comment:
Is this not already the default value used by the DrawerHeader (theme.primary)?

Reply:
This is by default using the theme.primary. This will be a case where we will have to apply the altTheme to both the Drawer and the DrawerHeader component. I'll update this accordingly.